packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -343,6 +372,12 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({
 
   const onError = (errors) => console.error(errors);
 
+  function handleRemoveImage(index: number) {
+    const updatedImages = [...uploadedImages];
```

Comment:
I don't _think_ it's necessary to do this. But if it is, I would suggest:
```suggestion
    updatedImages.splice(index, 1);
    setUploadedImages([...updatedImages]);
```

Or, alternatively (not sure we're supporting this syntax yet, but):

```ts
setUploadImages(updatedImages.toSpliced(index, 1));
```